dynamo/preprocessing/Preprocessor.py

```diff
@@ -627,19 +629,27 @@ def preprocess_adata_sctransform(
 
         temp_logger = LoggerManager.gen_logger("preprocessor-sctransform")
         temp_logger.log_time()
-        main_info("Applying Sctransform recipe preprocessing...")
+        main_info("Running Sctransform recipe preprocessing...")
 
         self.standardize_adata(adata, tkey, experiment_type)
-
         self._filter_cells_by_outliers(adata)
         self._filter_genes_by_outliers(adata)
+
+        self._calc_size_factor(adata)
+        self._normalize_by_cells(adata)
```

Comment:
Sctransform itself is like a kind of normalization. Not sure if we need this.

Reply:
Removed